packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -458,7 +458,7 @@ async function generateArmTempaltesFiles(ctx: CoreHookContext) {
     throw err(
       returnSystemError(
         new Error(`Failed to generate ${parameterEnvFileName} on migration`),
-        "Solution",
+        CoreSource,
```

Comment:
This is to replace `Teams Toolkit.FailedToGenerateArmTemplates` to `Solution.FailedToGenerateArmTemplates`. The changes in  these files is just to use a constant string `SolutionSource` to replace the string `"Solution"`. And here in *packages/fx-core/src/core/middleware/projectMigrator.ts* the changes is that the error is thrown by the core component instead of the solution component. It might be an issue and fixed in this pr. @wenytang-ms could you please review it? Thx!